crowd_anki/export/anki_exporter_wrapper.py

```diff
@@ -27,7 +28,7 @@ def __init__(self, collection,
         self.did = deck_id
         self.count = 0  # Todo?
         self.collection = collection
-        self.anki_json_exporter = json_exporter or AnkiJsonExporter(collection)
```

Comment:
alert - new instance is created. It's worrying that it's easy to make this error 🤔

Reply:
God damn it. Fixed :sweat_smile: